src/components/Homepage.js

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable global-require */
+/* eslint-disable max-len */
```

Comment:
- [ ] Let's don't disable linter rules. These rules are here to help us write the best code.